packages/path_provider/path_provider/example/pubspec.yaml

```diff
@@ -12,11 +12,11 @@ dev_dependencies:
     path: ../../../integration_test
   flutter_driver:
     sdk: flutter
-  pedantic: ^1.8.0
+  pedantic: ^1.10.0-nullsafety
 
 flutter:
   uses-material-design: true
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
Does it work to just revert all of the changes in `example/` rather than opting them all out explicitly (to make it easier to migrate it later)? That's what I've been doing so far and it seems to be working fine, but it may depend on whether there are version conflicts between the example and the plugin.

Reply:
Yea,  that works for me. I reverted them